### DISCUSSION TITLE
KaKario village world record and speedrun

### DISCUSSION THREAD

**Discussion by mmkkiinn** [nzp-team/nzportable#953] [Records and Speedruns]
My high round is 31 who can defeat me
![Screenshot_2024-10-28-09-37-15-60_2f85358b2198d26f8aca533d68bee793](https://github.com/user-attachments/assets/2b7a53e1-8d49-45d0-bcfd-bf6bc3088ac1)

**Comment by ToonLink0001** [comment]
did you get pap open, mine is bugged and the chickens wont collect

**Reply by mmkkiinn** [reply]
I open pap but hardly

**Comment by edgmaguilar-lab** [comment]
<img width="1612" height="720" alt="Screenshot_2026-07-18-10-42-57-38_2f85358b2198d26f8aca533d68bee793" src="https://github.com/user-attachments/assets/4fcc298a-2b7c-4f08-b8ed-7aac10feb132" />

My map got corrupted; I play on PSP and PC, but that save file is from the Android PSP version.
Lol

**Comment by edgmaguilar-lab** [comment]
<img width="1612" height="720" alt="Screenshot_2026-07-18-19-17-48-79_2f85358b2198d26f8aca533d68bee793" src="https://github.com/user-attachments/assets/290f9479-bbee-457d-b96c-d3384169ab05" />
gg
